src/pages/CreatePolicyWizard/CreatePolicyWizard.tsx

```diff
@@ -46,9 +46,9 @@ export const CreatePolicyWizard: React.FunctionComponent<CreatePolicyWizardProps
         return saveMutation.mutate(policy).then((res) => {
             if (res.payload?.type === 'Policy') {
                 if (policy.id === undefined) {
-                    addSuccessNotification('Created', `Policy "${policy.name}" created`);
+                    addSuccessNotification(`Created policy "${policy.name}"`, `From the Policies list, open ${policy.name}.`);
```

Comment:
From the blue text on the docs, I think they expect a link on the ${policy.name}.
This second argument on the addXNotification is actually a React.Node, so you can do stuff like:
```typescript
addSuccessNotification('blabla', <span> Bla bla <a href="#">{policy.name}</a></span>);
```